<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dersIslemleri</class>
 <widget class="QDialog" name="dersIslemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="dersler">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>571</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="ogrler">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>1111</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ders">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tüm Dersler</string>
   </property>
  </widget>
  <widget class="QLabel" name="ogr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Seçilen Derse Ait Öğrenciler</string>
   </property>
  </widget>
  <widget class="QLabel" name="baslik">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>20</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { color : red; }</string>
   </property>
   <property name="text">
    <string>Ders İşlemleri</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ders_kodu">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ders Kodu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ad">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ders Adı:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kod_lineEdit">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>90</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ad_lineEdit">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>170</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ekle">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>300</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Yeni Kayıt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>949</x>
     <y>300</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Güncelle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sil">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>390</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sil</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
